CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.1.1 (2019-06-25)
+
+- Fixed an issue where the button text was not returning to the original message after being clicked. [#15](https://github.com/blackbaud/skyux-lib-clipboard/pull/15)
```

Comment:
Suggested edit: Fixed an issue where the copy button text did not return to the original state after a user selected it.

What do you think @blackbaud-johnly ?

Reply:
https://github.com/blackbaud/skyux-lib-clipboard/pull/18 Moved the PR over to here